libcxx/src/include/overridable_function.h

```diff
@@ -85,13 +85,13 @@ _LIBCPP_HIDE_FROM_ABI bool __is_function_overridden(_Ret (*__fptr)(_Args...)) no
   uintptr_t __end   = reinterpret_cast<uintptr_t>(&__lcxx_override_end);
   uintptr_t __ptr   = reinterpret_cast<uintptr_t>(__fptr);
 
-#if defined(__arm64e__) && __has_feature(ptrauth_calls)
+#  if __has_feature(ptrauth_calls)
   // We must pass a void* to ptrauth_strip since it only accepts a pointer type. Also, in particular,
   // we must NOT pass a function pointer, otherwise we will strip the function pointer, and then attempt
   // to authenticate and re-sign it when casting it to a uintptr_t again, which will fail because we just
   // stripped the function pointer. See rdar://122927845.
```

Comment:
A lot of people get irrationally angry about rdar links, I'd suggest removing it ;)

Reply:
The comment is enough in itself to understand the issue. The rdar link is only adding more context in case you have access to it. Therefore I think the rdar link doesn't hurt, and I'll keep it.

Reply:
Oh I'm totally with you, but after a long discourse thread, others ended up with:
  https://llvm.org/docs/DeveloperPolicy.html#quality
`¯\_(ツ)_/¯`